docker/Dockerfile

```diff
@@ -30,6 +30,12 @@ RUN echo "source /opt/ros/humble/setup.bash" >> /home/$USER/.bashrc
 # Adds colcon autocomplete
 RUN echo "source /usr/share/colcon_argcomplete/hook/colcon-argcomplete.bash" >> /home/$USER/.bashrc
 
+# install gazebo11 classic
+RUN apt-get update \
```

Comment:
This is being executed as $USER, I suspect there should be a `sudo`

Reply:
Or moving this before the USER directive